app/src/main/java/chat/rocket/android/createchannel/ui/CreateChannelFragment.kt

```diff
@@ -213,8 +218,8 @@ class CreateChannelFragment : Fragment(), CreateChannelView, ActionMode.Callback
     private fun setupRecyclerView() {
         ui {
             recycler_view.layoutManager =
-                    LinearLayoutManager(context, LinearLayoutManager.VERTICAL, false)
-            recycler_view.addItemDecoration(DividerItemDecoration(it))
+                LinearLayoutManager(context, RecyclerView.VERTICAL, false)
```

Comment:
👀 👍

Reply:
For the usual `LinearLayoutManager`, vertical orientation and not reversed, the default constructor is enough -> `LinearLayoutManager(context)`